hash/nes.xml

```diff
@@ -66361,6 +66417,23 @@ We don't include these hacks because they were not burned into real carts nor so
 		</part>
 	</software>
 
+	<software name="toobink" cloneof="toobin">
+		<description>Toobin' (Korea)</description>
```

Comment:
Does this actually use the English name “Toobin'”?  If not, the description should be a transliteration of the Korean title (something like Tube-tamheom – it’s kind of annoying because the first half of it is the English loanword “tube”).

Reply:
In game it uses the English title. It's basically the US release with a Korean company splash screen tacked on before the title screen. Some packaging evidently uses the alt title, though I couldn't dig up any pics.